language-server/dm/document.ml

```diff
@@ -144,7 +144,7 @@ let compare_code_line x y =
 
 let code_lines_sorted_by_loc parsed =
   List.sort compare_code_line @@ List.concat [
-    (List.map (fun (_,x) -> Sentence x) @@ SM.bindings parsed.sentences_by_id) ;
+    (List.map (fun (_,x) -> Sentence x) @@ LM.bindings parsed.sentences_by_end) ;
```

Comment:
this should be reverted, or better the two maps should be printed for future debugging